frontend/src/pages/modelServing/screens/metrics/MetricsPageTabs.tsx

```diff
@@ -56,14 +60,15 @@ const MetricsPageTabs: React.FC = () => {
           <PerformanceTab />
         </Tab>
       )}
-      {biasMetricsEnabled && (
+      {biasMetricsEnabled && installed && (
```

Comment:
Question here: can we check installed in `useBiasMetricsEnabled`? if it's not installed, the biasMetricsEnabled will always be false, right?

Reply:
@DaoDaoNoCode I've just realised II forgot to hide the ModelBiasSettings section if the bias metrics feature flag is disabled. That means there will be no way to install it if the feature is disabled, so I don't think this should go in the feature flag hook.

I could create another hook for checking if it's enabled and installed? That would be a bit neater and make the logic reusable?

Reply:
Yes fair point. I think just use another hook to wrap `biasMetricsEnabled` and `installed` would be fine. And on the project details settings you can just hide/show it based on the `biasMetricsEnabled` hook.